src/Mod/TechDraw/Gui/DlgPrefsTechDrawDimensions.ui

```diff
@@ -559,6 +559,40 @@ Multiplier of 'Font Size'</string>
           </property>
          </widget>
         </item>
+        <item row="11" column="0">
+         <widget class="QLabel" name="label_18">
+          <property name="font">
+           <font>
+            <italic>true</italic>
+           </font>
+          </property>
+          <property name="toolTip">
+           <string>Controls the size of spacing between dimension line and dimension text.</string>
+          </property>
+          <property name="text">
+           <string>Line Spacing - ISO</string>
+          </property>
+         </widget>
+        </item>
+        <item row="11" column="2">
+         <widget class="Gui::PrefDoubleSpinBox" name="pdsbLineSpacingISO">
+          <property name="toolTip">
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controls the size of spacing between dimension line and dimension text. Value * linewidth is the line spacing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+          </property>
+          <property name="value">
+           <double></double>
+          </property>
+          <property name="prefEntry" stdset="0">
+           <cstring>LineSpacingISO</cstring>
+          </property>
+          <property name="prefPath" stdset="0">
+           <cstring>Mod/TechDraw/Dimensions</cstring>
+          </property>
```

Comment:
I have no idea what these lines define and if they are needed here (copied from GapASME in lines above)

Reply:
There are a bunch of widgets for updating preferences. This is how the "pref" widgets know where to write the parameter value.  Do you have the Designer plugin (https://wiki.freecad.org/Compile_on_Windows/en#Qt_Designer_plugin)?

Reply:
@WandererFan I have the designer plugin, but I avoided it as it wanted to reformat a lot of stuff in the UI file.

Reply:
The "pref" widgets create/update an entry in the FC parameter file (user.cfg). 

ADD_PROPERTY_TYPE macro adds a persistent property to an object.  The initial value of the property is set by looking up the value in user.cfg.  The look ups are centralized in Preferences/PreferencesGui.

Reply:
@WandererFan Right.

`          <property name="prefEntry" stdset="0">
           <cstring>LineSpacingISO</cstring>
          </property>
          <property name="prefPath" stdset="0">
           <cstring>Mod/TechDraw/Dimensions</cstring>
          </property>`
So this automatically sets the preferences. So I just have to add 2.0 to the ui file and we will always have a default preference in user.cfg where LineSpacingISO = 2.0?

Reply:
@WandererFan